src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -844,5 +844,50 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
             }
         }
         #endregion
+
+        #region Proxy tests
+        [Theory]
+        [MemberData(nameof(CredentialsForProxy))]
```

Comment:
A question: Do we need to add one test case for a bypassed Url?
At least in curlHandler, we have a different code path when ```Proxy``` object is set and [it is supposed to ignore the request uri](https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.EasyRequest.cs#L302)..